sup3r/bias/bias_calc.py

```diff
@@ -553,9 +553,15 @@ def _run_single(cls, bias_data, base_fps, bias_feature, base_dset,
                                         bias_feature, base_dset)
         return out
 
-    def fill_extend(self, out, smooth_extend):
+    def fill_smooth_extend(self, out, fill_extend=True, smooth_extend=0,
```

Comment:
nitpicky but fill and extend should be together in this title (we're filling the extension) or just "fill_and_smooth"

Reply:
Yeah I like either better lol